flask_server.py

```diff
@@ -98,7 +98,8 @@ def settings_page():
 @app.route("/gallery/<path:text>", methods=['GET'])
 def gallery(text):
     logger.info(f"GET request to /gallery/")
-    return render_template("gallery.html", title=text if text else "Gallery")
+    title = f"Gallery/{text}"
```

Comment:
"Gallery" + f"/{text}" if ... else "" and remove the condition from line 102?

Reply:
Done